src/collapse/collapse-content.tsx

```diff
@@ -0,0 +1,127 @@
+import React, {
+  useState,
+  useEffect,
+  useRef,
+  useContext,
+  useMemo,
+  useCallback,
+  PropsWithChildren
+} from 'react';
+import classNames from 'classnames';
+
+import dataTests from '../global/data-tests';
+
+import {getElementHeight, toPx} from './utils';
+import CollapseContext from './collapse-context';
+import {COLLAPSE_CONTENT_TEST_ID, COLLAPSE_CONTENT_CONTAINER_TEST_ID} from './consts';
+
+import styles from './collapse.css';
+
+const DURATION_FACTOR = 0.5;
+const DEFAULT_HEIGHT = 0;
+const VISIBLE = 1;
+const HIDDEN = 0;
+
+type Props = {
+  minHeight?: number;
+  className?: string;
+  'data-test'?: string | null | undefined;
+  disableAnimation?: boolean;
```

Comment:
@andrey-skl, Thank you for bringing this up. I was also uncertain about the decision and was considering passing these props to the same component. 
I think it would be better to pass these two props to the main <Collapse/> component. 

As for having a single prop, I'm not sure, I believe it would be better to have two different components.
I think it would make the API more obvious. WDYT?

Reply:
Two props for the main component would work I believe. Thanks!